examples/thermostat/thermostat-common/thermostat.zap

```diff
@@ -3057,6 +3057,21 @@
               "maxInterval": 65344,
               "reportableChange": 0
             },
+            {
+              "name": "OverrunCount",
+              "code": 12,
+              "mfgCode": null,
+              "side": "server",
+              "included": 1,
+              "storageOption": "RAM",
+              "singleton": 0,
+              "bounded": 0,
+              "defaultValue": "0x0000000000000000",
```

Comment:
Why are some of these expressed as 64 bits 0 value, but min/max interval is 1..65534? Many of the min/max here are wrong. The types are:

```
| 5  | BeaconLostCount        | uint32                              | _all_      | R       | N/A     | V      | ERRCNT      | V
| 6  | BeaconRxCount          | uint32                              | _all_      | R       | N/A     | V      | PKTCNT      | V
| 7  | PacketMulticastRxCount | uint32                              | _all_      | R       | N/A     | V      | PKTCNT      | V
| 8  | PacketMulticastTxCount | uint32                              | _all_      | R       | N/A     | V      | PKTCNT      | V
| 9  | PacketUnicastRxCount   | uint32                              | _all_      | R       | N/A     | V      | PKTCNT      | V
| 10 | PacketUnicastTxCount   | uint32                              | _all_      | R       | N/A     | V      | PKTCNT      | V
| 11 | CurrentMaxRate         | uint64                              | _all_      | R       | N/A     | V      | O           | V
| 12 | OverrunCount           | uint64                              | _all_      | R       | 0       | V      | ERRCNT      | V
```

^^^ Some 64 bits, most 32 bits. Many of the min/max intervals in these attributes imply a 16 bit range.

Reply:
The type and default values are defined in src/app/zap-templates/zcl/data-model/chip/wifi-network-diagnostics-cluster.xml 
wifi-network-diagnostics-cluster.xml

The min/max interval are auto-generated, for all unit64,   "minInterval": 1,  "maxInterval": 65534. What should be the correct internal value?

Reply:
This seems the bug in autogen, file https://github.com/project-chip/zap/issues/244 to track this. The final generated result does not match with its xml defination. For example

`<attribute side="server" code="0x0C" define="OVERRUN_COUNT" type="INT64U" min="0x0000000000000000" max="0xFFFFFFFFFFFFFFFF" writable="false" default="0x0000000000000000" optional="true">OverrunCount</attribute>`

The generated code is

        {
          "name": "OverrunCount",
          "code": 12,
          "mfgCode": null,
          "side": "server",
          "included": 1,
          "storageOption": "RAM",
          "singleton": 0,
          "bounded": 0,
          "defaultValue": "0x0000000000000000",
          "reportable": 0,
          "minInterval": 1,
          "maxInterval": 65534,
          "reportableChange": 0
        },